test/test_rules.py

```diff
@@ -93,3 +93,93 @@ def test_if_not_winner_is_winner_returns_false(self):
         current_board = ["1", "X", "3", "4", "X", "X", "7", "8", "9"]
         is_winner_true = rules.is_winner(current_board)
         self.assertEqual(False, is_winner_true)
+
+    def test_top_row_returns_top_row_of_board(self):
+        rules = Rules()
+        current_board = ["1", "2", "3", "4", "5", "6", "7", "8", "9"]
```

Comment:
can this be moved into a test setup?

Reply:
alternatively, can we test the rows in one test? ... even though there would be multiple asserts in one test, it would be testing just one concept.

Reply:
Got it!